### DISCUSSION TITLE
Time in Backup Settings not respecting timezone?

### DISCUSSION THREAD

**Discussion by Ender-Wang** [immich-app/immich#14026] [Q&A] [answered]
### The bug

Not really a bug, but sth I noticed. I am at CET, daylight saving ended last month, and it's UTC +1 now. I noticed that I have the default DB backup settings, which default to doing it at 2 a.m., but when I checked my DB backups, they were created at 3 a.m. I'm not sure if this is the expected behavior. Time on my server machine is correct.
![CleanShot_2024-11-08_23-39-12](https://github.com/user-attachments/assets/3b742ecc-66cc-4de2-88e1-7929130212fd)

### The OS that Immich Server is running on

Debian GNU/Linux 12 (bookworm) (raspberry pi 5, arm64)

### Version of Immich Server

1.120.1

### Version of Immich Mobile App

1.120.1 build.166

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
default
```

### Your .env content

```Shell
default
```

### Reproduction steps

Inspect the backup creation time.

**Comment by mmomjian** [comment, marked as answer]
The default env asks you to set your timezone, can you please try that.

**Reply by Ender-Wang** [reply]
Thanks, that should do the trick. But it would be great if immich pulls from system directly.

**Reply by bo0tzz** [reply]
Immich is inside a container, so it can't do that.
